index.md

```diff
@@ -3,7 +3,7 @@ layout: front
 title: Hello, We Are Convivio
 ---
 
-Convivio is a boutique agency designing and developing digital services for innovative public and private sector organisations.
+We do digital strategy, design, and development for humans.
```

Comment:
Steve suggests we loose the "for humans" as it doesn't stand on it's own without context and meaning behind.